### PR TITLE
Implement torrent-set-location call

### PR DESCRIPTION
**Pull request by Kriskras99**
I implemented the torrent-set-location call.
I also added VS Code and IntelliJ files to the gitignore and ran rustfmt to fix the inconsistent formatting.

If you don't like the formatting/gitignore changes you should be able to just merge the first commit.
Also when reviewing my torrent-set-location implementation it's probably best to look at the first commit.

Thank you for writing this awesome libary!

### CODE REVIEW

File: examples/session-get.rs [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+        password: env::var("TPWD")?,
+    };
     let client = TransClient::with_auth(&url, basic_auth);
     let response: Result<RpcResponse<SessionGet>> = client.session_get().await;
     match response {
         Ok(_) => println!("Yay!"),
         Err(_) => panic!("Oh no!")
     }
-    println!("Rpc reqsponse is ok: {}", response?.is_ok());
+    println!("Rpc response is ok: {}", response?.is_ok());
```
Comment (red-avtovo) [inline examples/session-get.rs:23]: Cheers for that :slightly_smiling_face:

**Comment by red-avtovo** [comment]
Hi @Kriskras99! Thank you for your PR and thanks again for that big format cleanup :smile: 
Feel free to bump the version to `0.3.2` and I will release a new version once this one will be merged

**Comment by Kriskras99** [comment]
Version bumped.

Happy new year!

**Review by red-avtovo — APPROVED** [review]
Great one! Thx again and Happy New Year! :tada:

**Pull request merged**